bokeh/models/ui/dialogs.py

```diff
@@ -20,53 +21,57 @@
 #-----------------------------------------------------------------------------
 
 # Bokeh imports
-from ....core.enums import TooltipAttachment
-from ....core.properties import Bool, Enum, Override
-from .html_annotation import HTMLAnnotation
+from ...core.properties import (
+    Bool,
+    Either,
+    Instance,
+    List,
+    NonNullable as Required,
+    Nullable,
+    String,
+)
+from ..dom import DOMNode
+from .ui_element import UIElement
 
 #-----------------------------------------------------------------------------
 # Globals and constants
 #-----------------------------------------------------------------------------
 
 __all__ = (
-    "Tooltip",
+    "Dialog",
 )
 
 #-----------------------------------------------------------------------------
 # General API
 #-----------------------------------------------------------------------------
 
-class Tooltip(HTMLAnnotation):
-    ''' Render a tooltip.
+Button = UIElement # TODO
 
-    .. note::
-        This model is currently managed by BokehJS and is not useful
-        directly from python.
-
-    '''
+class Dialog(UIElement):
+    """ """
 
     # explicit __init__ to support Init signatures
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
-    level = Override(default="overlay")
 
-    attachment = Enum(TooltipAttachment, help="""
-    Whether the tooltip should be displayed to the left or right of the cursor
-    position or above or below it, or if it should be automatically placed
-    in the horizontal or vertical dimension.
+    title = Nullable(Either(String, Instance(DOMNode)), default=None, help="""
     """)
```

Comment:
help string (here and below)

Reply:
Those are mostly stubs for the time being.

Reply:
I will write docstrings when those are at least partially implemented.